test/test_cli.py

```diff
@@ -9,7 +9,7 @@
 from wof.wofpy_config import makedirs
 
 
-@pytest.fixture
+@pytest.fixture(scope='function')
```

Comment:
I could not reproduce that locally nor on `Travis-CI`, but reducing the scope of the fixture is recommended anyways, so here it goes.